IdentityCore/src/MSIDError.m

```diff
@@ -99,10 +99,30 @@ MSIDErrorCode MSIDErrorCodeForOAuthError(NSString *oauthError, MSIDErrorCode def
     {
         return MSIDErrorServerAccessDenied;
     }
-    
+    if (oauthError && [oauthError caseInsensitiveCompare:@"tokenTransferFailedOTC"] == NSOrderedSame)
```

Comment:
Can we add a comment which explains a scenario (account transfer) and in which case (e.g., QR code or OTC is expired) this error gets returned by MSA? The same for other errors.